instrumentation/restlet/restlet-1.0/library/src/main/java/io/opentelemetry/instrumentation/restlet/v1_0/RestletServerSpanNaming.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.restlet.v1_0;
```

Comment:
How about moving it to a `io.opentelemetry.instrumentation.restlet.v1_0.internal` package? It needs to be public because javaagent uses it, but it shouldn't really belong to the public API of the library instrumentation (which should be minimal)